.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -37,4 +37,11 @@ Add any other context about the problem here.
 If possible, please include:
  - `kubectl describe` output
  - yaml definitions for your objects
- - log files for the operator and database pods
+ - The operator logs
+ - Your Custom Resource below assuming the pods are named `mongo-0`
```

Comment:
this sentence is not clear to what exactly you refer here

Reply:
see my change https://github.com/mongodb/mongodb-kubernetes-operator/pull/1483/commits/61b65eb993bcf9672e5a49f7abb60ec0a11dd8d0

Reply:
I see, added the changes